generic/ras_extended.py

```diff
@@ -175,6 +175,21 @@ def test1_vpdupdate(self):
             copyfile_path = os.path.join(self.outputdir, 'vpd.db')
             copyfile(path_db, copyfile_path)
             self.run_cmd("vpdupdate --path=%s" % copyfile_path)
+        if os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/var/lib/lsvpd/run.vpdupdate'
+        elif os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/run/run.vpdupdate'
+        process.run("mv %s /root/run.vpdupdate" % path)
```

Comment:
please use `shutil` python utility for file operation 
example can be 
`shutil.move(path, '/root/run.vpdupdate')`